pilot/pkg/serviceregistry/serviceentry/conversion.go

```diff
@@ -236,7 +236,7 @@ func convertServices(cfg config.Config) []*model.Service {
 }
 
 func convertEndpoint(service *model.Service, servicePort *networking.Port,
-	endpoint *networking.WorkloadEntry) *model.ServiceInstance {
+	endpoint *networking.WorkloadEntry, wleck *configKey) *model.ServiceInstance {
```

Comment:
Why even need configkey as we can get the workload name from endpoint

Reply:
I don't think name is there https://github.com/istio/api/blob/42be9dcd33d06aaa4c9d8f567a014f184a792648/networking/v1alpha3/workload_entry.pb.go#L258 Did I miss anything?

Reply:
IC, the one passed in is not Config